code/string_algorithms/src/palindrome_substring/palindrome_substring.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+/*
+ * Utilises 'iattempt' palindrome.c implementation.
+ * Part of Cosmos by OpenGenus Foundation.
+*/
+
+int isPalindromeIterative(char *input, int length)
```

Comment:
For all of your functions, the type goes on its own line. Also, you might want to read [this](https://stackoverflow.com/q/1921539/6525260).

Reply:
I have made the necessary changes but still its not passing the Travis-CI.

Reply:
@EvenCode You really don't need to worry about the Travis CI right now (It's still not fully complete). There are many files that don't pass currently. Just make sure yours does! (Check the build log to see if your file succeeded in running)